Doc/library/imaplib.rst

```diff
@@ -10,6 +10,7 @@
 .. changes for IMAP4_SSL by Tino Lange <Tino.Lange@isg.de>, March 2002
 .. changes for IMAP4_stream by Piers Lauder <piers@communitysolutions.com.au>,
    November 2002
+.. changes for IDLE by Forest <forestix@nom.one> August 2024
```

Comment:
```suggestion
.. changes for IMAP4 IDLE by Forest <forestix@nom.one> August 2024
```
(Avoid using just IDLE anywhere, since IDLE is also Pythons stdlib editing app -- I feel it's important to clarify this.)